apps/rush-lib/src/logic/InstallManager.ts

```diff
@@ -131,6 +134,7 @@ export class InstallManager {
   private _rushGlobalFolder: RushGlobalFolder;
   private _commonNodeModulesMarker: LastInstallFlag;
   private _commonTempFolderRecycler: AsyncRecycler;
+  private _maxInstallAttempts: number = 3;
```

Comment:
Intentionally increasing this?

Reply:
yes, I want to increase it to 3. I will retain the constant and set it to 3 as per your suggestion